_examples/mouse.go

```diff
@@ -104,7 +112,8 @@ func showMsg(g *gocui.Gui, v *gocui.View) error {
 
 func delMsg(g *gocui.Gui, v *gocui.View) error {
 	if err := g.DeleteView("msg"); err != nil {
-		return err
+		// Error removed for testing purpose, because delete could be called multiple times with the above keybindings
+		// return err
 	}
```

Comment:
Maybe we should not add the `if err := ...` here and move the comment to above the function call